src/main/java/uk/gov/hmcts/ccd/data/casedetails/CaseDetailsEntity.java

```diff
@@ -1,31 +1,43 @@
 package uk.gov.hmcts.ccd.data.casedetails;
```

Comment:
src/main/java/uk/gov/hmcts/ccd/data/casedetails/CaseDetailsEntity.java will be removed from the PR